spec/lib/miq_automation_engine/service_methods/miq_ae_service_manageiq-providers-vmware-infra_manager-provision_spec.rb

```diff
@@ -35,19 +35,18 @@ def invoke_ae
                                                     :provision_type => 'template',
                                                     :state => 'pending', :status => 'Ok',
                                                     :src_vm_id => @vm_template.id,
-                                                    :userid => @user.userid)
+                                                    :requester => @user)
         @miq_provision.miq_provision_request = @miq_provision_request
         @miq_provision.save!
         @miq_provision_request.save!
       end
 
       it "#miq_request" do
-        miq_request = @miq_provision_request.create_request
         method   = "$evm.root['#{@ae_result_key}'] = $evm.root['miq_provision'].miq_request"
         @ae_method.update_attributes(:data => method)
         ae_object = invoke_ae.root(@ae_result_key)
         ae_object.should be_kind_of(MiqAeMethodService::MiqAeServiceMiqRequest)
-        [:id].each { |meth| ae_object.send(meth).should == miq_request.send(meth) }
+        [:id].each { |meth| ae_object.send(meth).should == @miq_provision_request.send(meth) }
```

Comment:
Would be good to change this to just compare the `id` of the AR object and the service model object.  It would also remove the "Shadowing outer local variable" warning.